ed/csspatches/css-tables.json.patch

```diff
@@ -0,0 +1,26 @@
+From 093265b42f910145693bef023dc76fe2480311a7 Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Thu, 4 Mar 2021 15:50:25 +0100
+Subject: [PATCH] Drop zero-width spaces
+
+https://github.com/w3c/csswg-drafts/pull/4275
```

Comment:
since that PR was merged and reverted, it would probably best to have another reference (issue or pull request)

Reply:
I prepared a PR and updated the patch: https://github.com/w3c/csswg-drafts/pull/6077